pkgs/development/python-modules/grandalf/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyparsing
+, future
+, pytest
+, pytestrunner
+}:
+
+buildPythonPackage rec {
+  pname = "grandalf";
+  version = "0.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "7471db231bd7338bc0035b16edf0dc0c900c82d23060f4b4d0c4304caedda6e4";
+  };
+
+  propagatedBuildInputs = [
+    pyparsing
+    future
+  ];
+
+  # tests are not packaged as part of the source distribution
+  # however setup.py requires these dependencies
+  checkInputs = [ pytest pytestrunner ];
+
+  meta = with lib; {
+    description = "Grandalf is a python package made for experimentations with graphs and drawing algorithms";
```

```suggestion
    description = "A python package made for experimentations with graphs and drawing algorithms";
```
See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

---

Done.